README.md

```diff
@@ -21,7 +21,7 @@ Implemented API resources:
 - custom dimensions (with populators)
```

Comment:
BTW: this "development state" section should be removed (IMO)

Reply:
Agreed. Let's do it next time.